backend_services_deployment.md

```diff
@@ -75,3 +77,16 @@ podified OpenStack control plane services.
   ```
 
 ## Post-checks
+
+* Check if MariaDB PVC is bound:
+
+  ```
+  oc get pvc mariadb-openstack -o jsonpath='{.status.phase}{"\n"}'
+  ```
+
+  if not, run:
+
+  ```
+  # in install_yamls
+  make crc_storage
+  ```
```

Comment:
The `make crc_storage` bit is just for dev envs, but the post-checks would be better to be formulated for customer envs too. If we want to add a post-check as part of this PR, i'd add one that checks that the `mariadb-openstack` pod is running. That will cover both cases both for the PVC and other failure causes. And we don't need to add specific remedy info as root causes could vary.